content/2017/12/14/introducing-herman.rst

```diff
@@ -0,0 +1,417 @@
+that are not attached to a specific
+Sass/CSS objects will appear as free-floating prose,
+and can be given generic annotations --
+such as ``@group``, ``@example``, ``@link``, etc. --
+including the Herman design-token annotations.
+
+
+Additional markdown pages
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can add any number of `markdown documents`_
+to your style guide navigation,
+by defining ``extraDocs`` in your `Herman configuration`_.
+We use this to create an introduction to our design systems,
+provide quick access to a changelog,
+or document patterns that are not directly attached to Sass code.
+
+.. _markdown documents: /herman/docs/CONFIGURATION.html#extradocs
+.. _Herman configuration: /herman/docs/CONFIGURATION.html
+
+
+Third-party links
+^^^^^^^^^^^^^^^^^
+
+You can also add `external links`_
+to third-party dependencies -
+so all relevant documentation is accessible in one place.
+Define ``extraLinks`` in your Herman configuration,
+and we'll add links to the bottom of the sidebar.
+
+.. _external links: /herman/docs/CONFIGURATION.html#extralinks
+
+
+Open-source, and actively developed
+-----------------------------------
+
+We're using Herman on many of our production projects,
+and we have big plans for Herman's future.
+We'll keep adding features,
+and we invite you to do the same.
+The Herman code is `available on GitHub`_,
+and we'd love to hear your thoughts.
+File issues for bug-reports, feature requests,
+or help getting started --
+or send us your pull requests.
+
+We'll keep providing our own tutorials and demos
+based on the questions we hear most,
+but we'd also love to list any
+style guides or pattern librariea you build with Herman,
+or any `tutorials`_ you've written.
+
+We're always excited to collaborate,
+and we've provided some `"contributing" documentation`_
+to help you get involved.
+
+.. _available on GitHub: https://github.com/oddbird/sassdoc-theme-herman
+.. _tutorials: /herman/articles/
+.. _`"contributing" documentation`: /herman/docs/CONTRIBUTING.html
+
+
+Need help with your design system?
+----------------------------------
+
+At OddBird,
+we care deeply about test-coverage and documentation,
+accessibility, performance, and UX clarity.
+Herman is one part of our solution,
+but no tool can provide the magic bullet.
+
+**If you need help with a refactor** to
+improve design systems, performance, testing,
+documentation, and accessibility --
+or eliminate technical debt,
+and put better processes in place --
+we're here for you.
+Our team of experts
+can provide `a range of support, training, and consulting`_
+across the full stack of product design & development --
+helping find *the solution that best fits your team*.
+Don't hesitate to reach out.
+
+You can use our handy `contact form`_,
+talk to `@OddBird`_ on Twitter,
+or join our `public Slack`_
+(with a dedicated ``#herman`` channel).
+We're excited to hear from you!
+
+.. _contact form: /contact/
+.. _`@OddBird`: https://twitter.com/oddbird
```

Comment:
Is there a reason why this link needs backticks around it? Seems to work the same for me either way.